lms/envs/common.py

```diff
@@ -2677,3 +2677,6 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+# Grade book: max students per page
+MAX_STUDENTS_PER_PAGE_GRADE_BOOK = 25
```

Comment:
Any suggestion. I am using this number at 2 place. Should i create a constant in each app i.e ccx and instructor?
- instructor/ view/ api.py
- ccx/ view.py.